snuba/utils/streams/retries.py

```diff
@@ -0,0 +1,27 @@
+from abc import abstractmethod
```

Comment:
You could consider moving those modules that today are in `snuba/utils` into `streams/utils` As streams will not be in utils anymore once this goes into its own library.

Reply:
Sure, will rearrange after streams has been moved over, since there's a lot going on in there right now